index.html

```diff
@@ -0,0 +1,81 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <script  src="js/index.js"></script>
+    <meta charset="UTF-8">
+    <title>Fork Me! FCC: Test Suite Template</title>
+    <link rel="stylesheet" href="css/style.css">
+    <script src="https://cdn.freecodecamp.org/testable-projects-fcc/v1/bundle.js"></script>
+  </head>
+
+  <body>
+    <main id="main">
+      <h1 id="title">Nelson Mandela</h1>
+      <h3>Former President Of South Africa</h3>
+
+      <div id="img-div">
+        <img id = "image" src="images/nelson mandela.jpg" alt="a picture of Nelson Mandela">
+        <p id="img-caption">A picture of one of Africa's revoluttionaries. The late papa Madiba.</p>
+      </div>
+
```

Comment:
Please use semantic tags eg (Header) in your project, this will make it look professional.

Reply:
- [ ] Polite comment ✔️. Semantic tags also give context to screen readers and enable a vast audience of people with disabilities to use your website easily